lib/api/initiateMultipartUpload.js

```diff
@@ -143,8 +143,8 @@ function initiateMultipartUpload(authInfo, request, log, callback) {
             destinationBucket, log);
         if (backendInfoObj.err) {
             return process.nextTick(() => {
-                callback(backendInfoObj.err.customizeDescription('Location ' +
-                    'Constraint Info is invalid.'));
+                callback(backendInfoObj.err.customizeDescription(
```

Comment:
this is also done in put and copy I believe.  In any case, the customize step here should be taken out and just do it in the utility function.